server/db/migrations/20171020144702_initial_schema.js

```diff
@@ -0,0 +1,41 @@
+exports.up = knex =>
+	knex.schema
+		.raw('create extension if not exists "uuid-ossp"')
+		.createTable('flair', table => {
+			table.increments('id').primary().notNullable()
+			table.string('name', 20).notNullable()
+			table.string('description', 100)
+			table.text('filename').notNullable()
+		})
+		.createTable('ball', table => {
+			table.increments('id').primary().notNullable()
+			table.string('name', 20).notNullable()
+			table.string('description', 100)
+			table.text('filename').notNullable()
+		})
+		.createTable('player', table => {
```

Comment:
wins, losses?

Reply:
Yes... this needs discussion I think about what stats we're storing and in what format.